src/cmd/create.go

```diff
@@ -1,15 +1,10 @@
 package cmd
 
 import (
-	"io"
```

Comment:
Default Go formatter re-ordered inputs when I was updating references, we can remove these if not wanted.